WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -126,6 +180,25 @@ private extension DashboardViewController {
 extension DashboardViewController: DashboardUIScrollDelegate {
     func dashboardUIScrollViewDidScroll(_ scrollView: UIScrollView) {
         hiddenScrollView.updateFromScrollViewDidScrollEventForLargeTitleWorkaround(scrollView)
+        showOrHideSubtitle(offset: scrollView.contentOffset.y)
+    }
+
+    private func showOrHideSubtitle(offset: CGFloat) {
+        if ServiceLocator.stores.sessionManager.defaultSite?.name != nil {
```

Comment:
This is also a good fit for the early return pattern!

Reply:
Agree!